cmd/juju/model/setcredential.go

```diff
@@ -61,7 +61,7 @@ func (c *modelCredentialCommand) Info() *cmd.Info {
 	return &cmd.Info{
 		Name:    "set-credential",
 		Args:    "<cloud name> <credential name>",
-		Purpose: "Change cloud credential for the current or specified model.",
+		Purpose: "Links a remote credential to a model.",
```

Comment:
I do not like this wording:
1. It does not 'link". At the very best, it "re-links". I did not just pick a word out of the sky. Either "change" or "replace" or even "change reference"...?
2. It is not just "remote" credential. If needed, we do upload local credential.

Reply:
1.a Re-link: not always. You can get a remote credential that is not associated with any model (by first linking another credential to its model). You would then link that orphaned credential to a different model. Anyway, re-linking is still linking. :)
1.b The problem I see with the words "change a credential..." or "replace a credential..." is that it sounds like the content of the credential is changing or the content is being replaced, which isn't the case here. We're really just "pointing" a credential to a model.
2. It still always links a remote credential with a model. If we don't see the upload as an extra it gets unnecessarily complicated to summarise.

Reply:
Ahhh.